docs/tutorials/adapters.md

```diff
@@ -10,7 +10,7 @@ Final Cut 7 XML Format
 - <a href="https://developer.apple.com/library/content/documentation/AppleApplications/Reference/FinalCutPro_XML/AboutThisDoc/AboutThisDoc.html#//apple_ref/doc/uid/TP30001152-TPXREF101" target="_blank">Reference</a>
 
 Final Cut Pro X XML Format:
-- <a href="https://github.com/PixarAnimationStudios/OpenTimelineIO/issues/37">Status</a>
+- Status: Supported via the `fcp_xml` adapter
```

Comment:
Sorry, one more thing, this should be fcpx_xml, not fcp_xml, right?

```suggestion
- Status: Supported via the `fcpx_xml` adapter
```

Reply:
Yup, it should be fcpx_xml